continuous_integration/scripts/run_tests.sh

```diff
@@ -11,6 +11,6 @@ if [[ $COVERAGE == 'true' ]]; then
 fi
 
 echo "py.test dask --runslow $XTRATESTARGS"
-py.test dask --runslow $XTRATESTARGS
+py.test dask/tests/test_spark_compat.py --runslow $XTRATESTARGS
```

Comment:
This is just a temporary change for convenience. This should be reverted before merging this PR